.github/workflows/build-deb.yml

```
@@ -16,14 +16,23 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
 
+      - name: Compute deb package version
+        id: vars
+        run: |
```

there is nice syntax faciliating setting env var like this. Also, consider renaming `SHA` into sth more meaningful like
```suggestion
        env:
          COMMIT_SHA: ${{ github.sha }}
        run: |
```

---

Thanks for comments, I've included your suggestions in my latest comment (https://github.com/golemfactory/yagna/commit/3eccf0ae1a5c938fd1bb20d949b4bad8869038af)